.bazelci/presubmit.yml

```diff
@@ -4,22 +4,48 @@ tasks:
   ubuntu1604:
     name: ":linux:"
     shell_commands:
-    - yes | ./configure
+    - wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+    - bash ./Miniconda3-latest-Linux-x86_64.sh -b -p $HOME/miniconda
+    - source $HOME/miniconda/bin/activate
+    - wget https://raw.githubusercontent.com/plaidml/plaidml/master/environment.yml
+    - $HOME/miniconda/bin/conda env update -f ./environment.yml
```

Comment:
There's no need to create a conda environment on unix, bazel will manage all of its conda environments directly.

Reply:
Fixed in commit [`814cd24`](https://github.com/plaidml/plaidml/pull/435/commits/814cd246c8e9fee003c7abba4b074f7e7dc9259d)